skins/laika/src/store/address/index.ts

```diff
@@ -37,6 +37,7 @@ export default function (): Module<AddressState, any> {
 			email: Validity.INCOMPLETE,
 			addressType: Validity.VALID,
 		},
+		requiredFields: requiredFields,
```

Comment:
a nitpick but this should probably go up with the rest of the single lined state variables.